src/app/tests/suites/certification/Test_TC_DRLK_2_6.yaml

```diff
@@ -107,40 +107,38 @@ tests:
           error: INVALID_COMMAND
 
     - label:
-          "Step 5: TH sends Get Holiday Schedule Command to DUT with Invalid
-          HolidayIndex as 15"
+          "Step 5: TH sends GetHolidaySchedule Command to DUT with Invalid
+          HolidayIndex > number_holiday_sch_supported ."
       PICS: DRLK.S.F0b && DRLK.S.C12.Rsp && DRLK.S.C12.Tx
       command: "GetHolidaySchedule"
       arguments:
           values:
               - name: "HolidayIndex"
-                value: 15
+                value: NumberOfHolidaySchedulesSupportedValue + 1
       response:
           values:
-              - name: "HolidayIndex"
-                value: 15
               - name: "Status"
                 value: 133
 
     - label:
-          "Step 6: TH sends Get Holiday Schedule Command to DUT with the
-          HolidayIndex as 10 (value is in the the range of step 1 but Holiday
-          Schedule entry not available)"
+          "Step 6: TH sends GetHolidaySchedule Command to DUT with the
+          HolidayIndex as any value that is different from 1 and value <=
+          number_holiday_sch_supported"
       PICS: DRLK.S.F0b && DRLK.S.C12.Rsp && DRLK.S.C12.Tx
       command: "GetHolidaySchedule"
       arguments:
           values:
               - name: "HolidayIndex"
-                value: 10
+                value: NumberOfHolidaySchedulesSupportedValue
       response:
           values:
               - name: "HolidayIndex"
-                value: 10
+                value: NumberOfHolidaySchedulesSupportedValue
               - name: "Status"
                 value: 139
```

Comment:
```suggestion
                value: 0x8B
```

Reply:
@bzbarsky-apple  yaml script is updated as per the suggestion.